Source/UI/Controllers/CardViewController.swift

```diff
@@ -101,6 +108,10 @@ public class CardViewController: UIViewController,
     /// Called after the controller's view is loaded into memory.
     override public func viewDidLoad() {
         super.viewDidLoad()
+        UIFont.loadAllCheckoutFonts
+        UITextField.disableHardwareLayout()
```

Comment:
`UITextField.disableHardwareLayout()` , please review my last comment.
`UIFont.loadAllCheckoutFonts` is to register all fonts in SDK.

Reply:
If we want to register all the fonts in the SDK, is the viewDidLoad of one screen the right place to do it? Feels like a lifecycle event for the SDK, rather than one screen. Do we have some form of SDK initialiser, or place where we prepare the start of the UI where we could do this instead?